web_src/css/install.css

```diff
@@ -1,5 +1,6 @@
-.page-content.install {
-  padding-top: 45px;
+.page-content.install .install-config-container {
+  max-width: 900px;
+  margin: auto;
```

Comment:
Do we need auto margin in both directions or just one?

Reply:
Both